Formula/libpaho-mqtt.rb

```diff
@@ -0,0 +1,32 @@
+class LibpahoMqtt < Formula
+  desc "Eclipse Paho C client library for MQTT"
+  homepage "https://eclipse.github.io/paho.mqtt.c/"
+  url "https://github.com/eclipse/paho.mqtt.c/archive/refs/tags/v1.3.12.tar.gz"
+  sha256 "6a70a664ed3bbcc1eafdc45a5dc11f3ad70c9bac12a54c2f8cef15c0e7d0a93b"
+  license "EPL-2.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", "-DBUILD_SHARED_LIBS=ON", *std_cmake_args
```
Unless it already builds shared libraries, but it doesn't look like it does by default.

Reply:
it does build shared libs by default

```
$ tree /opt/homebrew/Cellar/libpaho-mqtt/1.3.12/
/opt/homebrew/Cellar/libpaho-mqtt/1.3.12/
├── INSTALL_RECEIPT.json
├── LICENSE
├── README.md
├── about.html
├── bin
│   └── MQTTVersion
├── include
│   ├── MQTTAsync.h
│   ├── MQTTClient.h
│   ├── MQTTClientPersistence.h
│   ├── MQTTExportDeclarations.h
│   ├── MQTTProperties.h
│   ├── MQTTReasonCodes.h
│   └── MQTTSubscribeOpts.h
├── lib
│   ├── cmake
│   │   └── eclipse-paho-mqtt-c
│   │       ├── eclipse-paho-mqtt-cConfig-release.cmake
│   │       ├── eclipse-paho-mqtt-cConfig.cmake
│   │       └── eclipse-paho-mqtt-cConfigVersion.cmake
│   ├── libpaho-mqtt3a.1.3.12.dylib
│   ├── libpaho-mqtt3a.1.dylib -> libpaho-mqtt3a.1.3.12.dylib
│   ├── libpaho-mqtt3a.dylib -> libpaho-mqtt3a.1.dylib
│   ├── libpaho-mqtt3c.1.3.12.dylib
│   ├── libpaho-mqtt3c.1.dylib -> libpaho-mqtt3c.1.3.12.dylib
│   └── libpaho-mqtt3c.dylib -> libpaho-mqtt3c.1.dylib
├── notice.html
└── share
    └── doc
        └── Eclipse Paho C
            ├── CONTRIBUTING.md
            ├── README.md
            ├── edl-v10
            ├── epl-v20
            ├── notice.html
            └── samples
                ├── MQTTAsync_publish.c
                ├── MQTTAsync_publish_time.c
                ├── MQTTAsync_subscribe.c
                ├── MQTTClient_publish.c
                ├── MQTTClient_publish_async.c
                ├── MQTTClient_subscribe.c
                ├── paho_c_pub.c
                ├── paho_c_sub.c
                ├── paho_cs_pub.c
                ├── paho_cs_sub.c
                └── pubsub_opts.c

10 directories, 38 files
```

But happy to add one.

Reply:
No need if it builds shared libraries by default.